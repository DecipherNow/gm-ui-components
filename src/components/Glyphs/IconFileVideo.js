import React from "react";
import Icon from "../Icon";

export default function FileVideo(props) {
  return (
    <Icon {...props} glyphName="FileVideo">
      <path
        d="M6.49999998,3.99999997 L16,3.99999997 L18,5.99999997 L18,19.5 C18,19.7761424 17.7761424,20 17.5,20 L6.49999998,20 C6.22385761,20 5.99999997,19.7761424 5.99999997,19.5 L5.99999997,4.49999997 C5.99999997,4.2238576 6.22385761,3.99999997 6.49999998,3.99999997 Z M10.7922159,9.45947399 C10.7071096,9.39817727 10.6048826,9.36519544 10.5,9.36519544 C10.2238576,9.36519544 9.99999999,9.58905307 9.99999999,9.86519544 L9.99999999,15.2459011 C9.99999999,15.3507837 10.0329818,15.4530107 10.0942785,15.538117 C10.2556649,15.7621907 10.5681421,15.8130089 10.7922159,15.6516226 L14.5275842,12.9612697 C14.5712855,12.9297944 14.6096145,12.8914654 14.6410898,12.8477641 C14.8024762,12.6236904 14.751658,12.3112132 14.5275842,12.1498268 L10.7922159,9.45947399 Z"
        id="File-Video"
         
        fillRule="nonzero"
      />
    </Icon>
  );
}

FileVideo.displayName = "FileVideo";
