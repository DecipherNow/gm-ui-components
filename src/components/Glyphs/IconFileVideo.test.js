import React from "react";
import { shallow } from "enzyme";

import IconFileVideo from "./IconFileVideo.svg";

describe("IconFileVideo", () => {
  it("matches snapshot", () => {
    const aIconFileVideo = shallow(<IconFileVideo />);
    expect(aIconFileVideo).toMatchSnapshot();
  });
});