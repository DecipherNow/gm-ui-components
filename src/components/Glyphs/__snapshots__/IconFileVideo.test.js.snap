// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconFileVideo matches snapshot 1`] = `<IconMock />`;
